payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AddressElement.kt

```diff
@@ -230,3 +226,47 @@ open class AddressElement constructor(
         this.rawValuesMap = rawValuesMap
     }
 }
+
+internal suspend fun updateLine1WithAutocompleteAffordance(
```

Comment:
I won't claim this to be my finest moment. But this was the best place I could find to do it. And it is tested.